src/context/product-context.js

```diff
@@ -29,7 +29,7 @@ const ProductProvider = ({ children }) => {
       (async () => {
         try {
             const response = await axios.get("/api/products");
```

Comment:
```
const {data} = await axios.get(....)
```
You can destructure the data form response so that you don't need to use response.data.